README.md

```diff
@@ -1 +1,44 @@
-# docker-platforms
+# Docker-platforms
+
+### 1 - Before running any platform
+
+Create the default network that will be used by traefik
+```bash
+docker network create web
+```
+### 2 - Starting up Traefik
+
+Make sure you have created the default external network
+```bash
+	cd traefik
+	docker-compose up -d
```

Comment:
Assuming that I don't want to run Traefik - what do I need to do to run Magento 2?

Reply:
- Create the networks as usual
- Expose port 80 in the Magento docker-compose file

I'll write about this

Reply:
I would then need to shut my Play wouldn't I? I cannot do that. Can we change the ports?

Reply:
For the time being, yes. If you'd like to change the ports, do so on traefik, exposing 80 to some other free port you have.
You need to update the platform hostname as well.

Reply:
I'm a bit lost. Any examples?

Reply:
Updated to use 8081 by default